samplingsimulatorpy/generate_virtual_pop.py

```diff
@@ -9,7 +9,7 @@ def generate_virtual_pop(size, distribution_func, *para):
     Parameters
     ----------
     size : int
-        The size of the virtual population 
+        The size of the virtual population
     distribution_func : func
         The function that came from numpy.random
     *para : int
```

Comment:
Hey Tao, would it be possible to add an extra parameter so that someone can pass in a name for the variable they are generating (similar to the R file)?

Reply:
You are right! I will update it